test/testcanvas.go

```diff
@@ -162,7 +162,7 @@ func (c *testCanvas) Capture() image.Image {
 // NewCanvas returns a single use in-memory canvas used for testing
 func NewCanvas() WindowlessCanvas {
 	padding := fyne.NewSize(10, 10)
-	return &testCanvas{size: padding, padded: true, scale: 1.0}
+	return &testCanvas{size: fyne.NewSize(100, 100).Add(padding).Add(padding), padded: true, scale: 1.0}
```

Comment:
I don't understand this change - it's now 120*120 from 10x10?

Reply:
Yes. The former size was too small for the most widgets and thus affected the position adjustment of pop-ups. That's why I increased the space. I added the padding size, just to be sure that it doesn't reduce the expected available space of at least 100x100.

Reply:
This doesn’t seem the right approach, as ensuring it is at least content.MinSize() after layout ensures the correct content. By making it 100x100 any components that are normally smaller will be expanded unexpectedly... won’t they?

Reply:
In a “normal” environment, yes. I'm not sure about the test environment.
However, with 10x10 it made the pop-up placement untestable.

Reply:
Can’t you enlarge the canvas in the test?